Source/CLI/CLI_Help.cpp

```diff
@@ -21,12 +21,12 @@ return_value Help(const char* Name, bool Full)
     "Usage: \"" << Name << " [-Options...] FileName1 [Filename2...]\"\n";
     if (!Full)
     {
-        cout << "\"" << Name << " --Help\" for displaying more information.\n";
+        cout << "\"" << Name << " --help\" for displaying more information.\n";
         return ReturnValue_OK;
     }
     cout << "\n"
     "Options:\n"
-    "--Help, -h         Display this help and exit.\n"
+    "--help, -h         Display this help and exit.\n"
```

Comment:
I'd prefer if the options were either uniformly lowercase or uppercase, but this change makes --help lowercase while leaving --Version as uppercase. I prefer lowercase overall, but I know MediaArea products tend to use uppercase. So I don't care what is chosen here, as long as it remains consistent.

Reply:
+1 for what @ablwr said

Reply:
Actually `--version` is already [lowercase](https://github.com/mipops/dvrescue/blob/master/Source/CLI/CommandLine_Parser.cpp#L35), this PR puts everything in lowercase.
I fixed the misleading help part.